cache_flat_mutation_reader.hh

```diff
@@ -832,7 +832,8 @@ inline
 void cache_flat_mutation_reader::add_clustering_row_to_buffer(mutation_fragment_v2&& mf) {
     clogger.trace("csm {}: add_clustering_row_to_buffer({})", fmt::ptr(this), mutation_fragment_v2::printer(*_schema, mf));
     auto& row = mf.as_clustering_row();
-    auto new_lower_bound = position_in_partition::after_key(row.key());
+    auto new_lower_bound = position_in_partition(
+            position_in_partition_view::after_key(*_schema, row.key()).view);
```

Comment:
Why not use `position_in_partition::after_key()` here?

Reply:
Yes, we could.